A32NX/html_ui/Pages/A32NX_Core/A32NX_FuelPred.js

```diff
@@ -207,26 +207,14 @@ class A32NX_FuelPred {
         const flightLevelMatrix = _buildBMatrix6(flightLevel);
         const mmOfDistFL = math.multiply(flightLevelMatrix, airDistanceMatrix);
         //TODO Create logic for handling 200NM and FL390 = 0
-        if (!alternate) {
-            switch (computation) {
-                case this.computations.FUEL:
-                    const deviation = 0;
-                    return (Math.round(math.sum(math.dotMultiply(fuelConsumedCoeff,mmOfDistFL)))) + deviation;
-                case this.computations.TIME:
-                    return (Math.round(math.sum(math.dotMultiply(timeCoeff,mmOfDistFL))));
-                case this.computations.CORRECTIONS:
-                    return (Math.round(math.sum(math.dotMultiply(correctionsCoef,mmOfDistFL))));
-            }
-        } else {
-            switch (computation) {
-                case this.computations.FUEL:
-                    const deviation = 0;
-                    return (Math.round(math.sum(math.dotMultiply(altFuelConsumedCoef,mmOfDistFL)))) + deviation;
-                case this.computations.TIME:
-                    return (Math.round(math.sum(math.dotMultiply(altTimeCoef,mmOfDistFL))));
-                case this.computations.CORRECTIONS:
-                    return (Math.round(math.sum(math.dotMultiply(altCorrectionsCoeff,mmOfDistFL))));
-            }
+        switch (computation) {
+            case this.computations.FUEL:
+                const deviation = 0;
+                return (Math.round(math.sum(math.dotMultiply((!alternate ? fuelConsumedCoeff : altFuelConsumedCoef) ,mmOfDistFL)))) + deviation;
+            case this.computations.TIME:
+                return (Math.round(math.sum(math.dotMultiply((!alternate ? timeCoeff : altTimeCoef),mmOfDistFL))));
+            case this.computations.CORRECTIONS:
+                return (Math.round(math.sum(math.dotMultiply((!alternate ? correctionsCoef : altCorrectionsCoeff),mmOfDistFL))));
```

Comment:
```suggestion
                return (Math.round(math.sum(math.dotMultiply((alternate ? altFuelConsumedCoef : fuelConsumedCoeff ) ,mmOfDistFL)))) + deviation;
            case this.computations.TIME:
                return (Math.round(math.sum(math.dotMultiply((alternate ? altTimeCoef : timeCoeff ),mmOfDistFL))));
            case this.computations.CORRECTIONS:
                return (Math.round(math.sum(math.dotMultiply((alternate ? altCorrectionsCoeff : correctionsCoef ),mmOfDistFL))));
```

Reply:
I'd say that makes stuff easier to read and there is no brain fuck with true being false and false being true